main/lsp/LSPTypechecker.cc

```diff
@@ -553,6 +550,8 @@ LSPTypechecker::SlowPathResult LSPTypechecker::runSlowPath(LSPFileUpdates &updat
         pipeline::unpartitionPackageFiles(indexed, std::move(nonPackagedIndexed));
         // TODO(jez) At this point, it's not correct to call it `indexed` anymore: we've run namer too
 
+        this->cacheOpenFiles(indexed, std::move(openFiles), workers);
```

Comment:
This call is after the call to `pipeline::name`. Before, the `indexedFinalGS` map was populated with the result of `indexAndComputeFileHashes`, which only runs up to `indexOne`.

Reply:
Great catch, I'll move this above the namer!